packages/web-components/src/components/global-header/components/global-header/src/components/HybridIpaasHeaderReactWrapper/HybridIpaasHeaderReactWrapper.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/* eslint jsdoc/require-jsdoc: 0 */
+
+import React from 'react';
+import { createComponent } from '@lit/react';
+
+import { HybridIpaasHeader as WCHybridIpaasHeader } from '../HybridIpaasHeader/HybridIpaasHeader';
+import { ReactWrapperProps } from '../../types/Header.types';
+
+const AI_CALLBACK_EVENT = 'hybrid-ipaas-header-ai-callback';
```

Comment:
might be safer to prefix the event names with "clabs-" to ensure they dont clash with anything else